terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf

```
@@ -139,3 +137,31 @@ resource "kubernetes_config_map" "mattermost_performance_monitoring_per_cluster"
     "mattermost_performance_monitoring_per_cluster.json" = "${file("../../../../grafana-dashboards/mattermost_performance_monitoring_per_cluster.json")}"
   }
 }
+
+resource "kubernetes_config_map" "skydns" {
+  metadata {
+    name      = "skydns-dashboard"
+    namespace = "monitoring"
+    labels = {
+      cloud_dashboards = "skydns"
+    }
+  }
+
+  data = {
+    "coredns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
```

this one as well needs to be updated
```suggestion
    "skydns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
```

---

Oops, typo! Nice catch @angeloskyratzakos !!!